Containerfile

```diff
@@ -37,14 +33,18 @@ LABEL org.opencontainers.image.authors="https://issues.redhat.com/browse/THREESC
       # org.opencontainers.image.created=""
 
 ARG PORTA_IMAGE=quay.io/3scale/porta:nightly
-COPY --from=builder /home/rpms /tmp/rpms
-COPY --from=builder /home/searchd /
-COPY --from=$PORTA_IMAGE /opt/system/config/standalone.sphinx.conf "/etc/sphinx/system.sphinx.conf"
-RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release && \
-    rpm -iv --excludedocs /tmp/rpms/* && \
-    rm -rf /tmp/rpms && \
-    mkdir -p /var/lib/searchd /var/run/sphinx && \
-    chmod 770 /var/lib/searchd /var/run/sphinx
+COPY --from=builder /tmp/manticore_uselessly_long_path_to_prevent_rpm_build_issues/build/*.rpm /tmp/rpms/
+COPY --from=$PORTA_IMAGE /opt/system/config/standalone.sphinx.conf "/etc/manticoresearch/manticore.conf"
+ENV MANTICORE_RPMS="manticore-converter* manticore-icudata* manticore-common* manticore-server-core* manticore-server*"
+RUN microdnf install -y --nodocs mysql openssl boost-context boost-filesystem zlib && \
+    cd /tmp/rpms && ls -l && \
+    rpm -iv --excludedocs $MANTICORE_RPMS && \
+    cd - && rm -rf /tmp/rpms && \
+    microdnf clean all && \
+    sed -i -e 's#/var/run/sphinx/#/var/run/manticore/#' -e 's#/var/lib/searchd#/var/lib/manticore#' /etc/manticoresearch/manticore.conf && \
```

Comment:
I guess this is temporary, and when it's all finalized and deployed, we can just update the paths in the productized build?
Maybe we can leave a `#TODO` here for this task?

Reply:
Would be nice to do at some point. I don't consider it a big issue though. If you find it important I can add a TODO.

Reply:
Yeah, I think it would be nice to add a TODO, so we don't forget to do this eventually.
Not very important indeed, but it would be cleaner.